ci-operator/config/openshift/knative-eventing/openshift-knative-eventing-release-v1.2__49.yaml

```diff
@@ -649,6 +649,93 @@ tests:
           cpu: 100m
       timeout: 4h0m0s
     workflow: generic-claim
+- as: reconciler-aws-ocp-49-continuous
```

Comment:
It would be nice if this new section would be added to the periodics genrator:

https://github.com/openshift/knative-eventing/blob/main/openshift/ci-operator/generate-ci-config.sh